index.js

```diff
@@ -12,11 +12,10 @@ module.exports = {
   // ],
 
   locals(options) {
-    let name = options.entity.name;
-    let component = componentize(name);
+    let name = componentize(options.entity.name);
```

Comment:
`componentize` adds `-app` if the name of the project was only a single word, we don't need that here.

Instead we should do:

```js
let name = dasherize(options.entity.name);
let component = componentize(name);
let className = stringUtils.classify(options.entity.name);
return { name, className, component };
```